examples/src/main/java/io/dapr/examples/workflows/DemoWorkflowService.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2023 The Dapr Authors
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package io.dapr.examples.workflows;
+
+import com.microsoft.durabletask.DurableTaskClient;
+import com.microsoft.durabletask.DurableTaskGrpcClientBuilder;
+import com.microsoft.durabletask.NewOrchestrationInstanceOptions;
+import io.dapr.workflows.runtime.WorkflowRuntime;
+
+import java.util.concurrent.TimeUnit;
+
+/**
+ * For setup instructions, see the README.
+ */
+public class DemoWorkflowService {
+
+  /**
+   * The main method of this app.
+   *
+   * @param args The port the app will listen on.
+   * @throws Exception An Exception.
+   */
+  public static void main(String[] args) throws Exception {
+    // Register the Workflow with the runtime.
+    WorkflowRuntime.getInstance().registerWorkflow(DemoWorkflow.class);
+    WorkflowRuntime.getInstance().start();
+    System.out.println("Start workflow runtime");

```

Comment:
Since `start()` is non-blocking, won't the program exit immediately after we start the runtime?

Reply:
That hasn't been my experience running this example code. The new thread created by `start()` continues to run until it receives an interrupt.